Prepare integration branch for 2.0 general availability (#513)
ably/ably-python: README.md

## Overview

> This is a Python client library for Ably. The library currently targets the [Ably 2.0 client library specification](https://ably.com/docs/client-lib-development-guide/features).

Reviewer: Probably worth updating this to avoid the redirect now.

```suggestion
This is a Python client library for Ably. The library currently targets the [Ably 2.0 client library specification](https://sdk.ably.com/builds/ably/specification/main/features/).
```

await client.close()
```

## Using the realtime client

> ### Creating a client using API key

Reviewer: ```suggestion
### Create a client using an API key
```

    client = AblyRealtime('api:key')
```

> ### Create a client using an token auth

Reviewer: ```suggestion
### Create a client using token auth
```
